recipe/meta.yaml

```
@@ -1,55 +1,53 @@
-{% set version = "1.4.4" %}
+{% set name = "panel" %}
+{% set version = "1.5.1" %}
 
 package:
-  name: panel
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/p/panel/panel-{{ version }}.tar.gz
-  sha256: 659e9fc5b495e6519c5d07e8148fa5eeed9bc648356ec83fc299381ba5a726ef
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/panel-{{ version }}.tar.gz
+  sha256: 8e5acf893997c124678adbea5e8fcf47a1b40fa22aeae1bc0094f20291b0cfc0
 
 build:
   number: 0
   # nodejs currently isn't available on s390x
-  skip: True  # [py<39 or s390x]
+  skip: True  # [py<310 or s390x]
   entry_points:
     - panel = panel.command:main
   script:
     - {{ PYTHON }} -m pip install . -vv  --no-deps --no-build-isolation
 
 requirements:
   host:
-    - packaging
     - python
-    - pip
-    - setuptools
-    - wheel
-    # These are also needed at build time.
-    - bokeh >=3.4.0,<3.5
-    - markdown
-    - pyct >=0.4.4
-    - param >=2.0.0,<3
-    - pyviz_comms >=2.0.0
-    - nodejs 18.16
+    - hatchling
+    - hatch-vcs
+    - param >=2.1.0
+    - bokeh >=3.5.0,<3.7.0
+    - pyviz_comms >=0.7.4
+    - nodejs 20.17
     - requests
-    - tqdm >=4.48.0
+    - packaging
+    - pip
   run:
     - python
-    - bokeh >=3.4.0,<3.5
-    - param >=2.1.0,<3
+    - bokeh >=3.5.0,<3.7.0
+    - param >=2.1.0,<3.0
     - pyviz_comms >=2.0.0
     - markdown
     - markdown-it-py
     - linkify-it-py
     - mdit-py-plugins
     - requests
-    - tqdm >=4.48.0
     - bleach
-    - typing_extensions
+    - typing-extensions
     - pandas >=1.2
     - packaging
+    - tqdm
   run_constrained:
     - holoviews >=1.18.0
+    - bokeh-fastapi ==0.1.0
```

bokeh-fastapi was pretty broken, this would be better:

```suggestion
    - bokeh-fastapi ==0.1.1
```